src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -240,6 +242,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		return this._kernelAliases;
 	}
 
+	public get currentKernelAlias(): string {
+		return this._currentKernelAlias;
+	}
+
+	public get currentKernel(): string {
+		return this._currentKernel;
```

Comment:
Hmm I still don't love adding a currentKernel property here

Reply:
The reason I have this is mainly due to the scenario when a user connects to a Kusto Connection. The default kernel will be SQL as a result we have the changeKernel method when the loadAttachtoDropdown gets called, once the currentKernel gets updated then it allows us to keep track of the when the kernel is also Kusto. 
Since there is no other property that we can look for when the current kernel is Kusto, or in the future another notebookKernelAlias. I tried relying on the currentKernelAlias property however that raised issues with the getKernelDisplayName, not getting the appropriate display name when the connection is Kusto, but the user is changing kernels (to python for example).